src/dom_util.js

```diff
@@ -68,7 +68,7 @@ const getRoot = function(node) {
  *     corresponding to node, if present.
  */
 const getActiveElement = function(node) {
-  const root = getRoot(node);
+  const root = getRootNode.call(node);
```

Comment:
It looks like once you control for megamorphic access, the difference goes away (which you will hit for anything that sees more than I think 4 types?).

https://jsperf.com/getrootnode-vs-call-megamorphic